www/docs/api/key.php

```diff
@@ -10,29 +10,39 @@
 $this_page = 'api_key';
 $PAGE->page_start();
 $PAGE->stripe_start();
-echo '<p>TheyWorkForYou API calls require a key, so that we can monitor usage
+?>
+
+<h3>About API Keys</h3>
+<p>TheyWorkForYou API calls require a key, so that we can monitor usage
 of the service, and provide usage stats to you. Please see <a href="/api">the
-API overview</a> for how to use your key.';
+API overview</a> for how to use your key.<br/><br/>
+
+<?php
 if ($THEUSER->loggedin()) {
     if (get_http_var('create_key') && get_http_var('reason')) {
-        create_key(get_http_var('commercial'), get_http_var('reason'));
+        $estimated_usage = intval(get_http_var('estimated_usage'));
```

Comment:
The recommendation is that if you don't need bases other than 10 you should do this with `(int) get_http_var('estimated_usage')` instead of `intval()`.